mender-convert-package

```diff
@@ -212,7 +215,9 @@ disk_create_file_system_from_folder "work/rootfs/" "work/rootfs.img" \
 log_info "Copying root filesystem image to deploy directory"
 run_and_log_cmd "cp --sparse=always work/rootfs.img deploy/${image_name}.${image_fs_type}"
 
-mender_create_artifact "${device_type}" "${artifact_name}" "${image_name}"
+if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
+    mender_create_artifact "${device_type}" "${artifact_name}" "${image_name}"
+fi
```

Comment:
I understand why we do this: an Artifact without mender in it is going to fail.

It somehow changes the interface of the whole utility, I suggest we add here an INFO being explicit: `Skipping image Artifact generation, mender Client not installed` (or something like that...)

Reply:
I just wanted to get a successful run...

The reason being:

```
if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
    artifact_name=$(cat work/rootfs/etc/mender/artifact_info | sed 's/[^=]*=//')
fi
```
When we don't install the client, the `artifact_info` file does not exist, so we either have to put it back in, or give the artifact name a standard name. 

Also, what is the artifact needed for when it does not contain  a client?